### PR TITLE
Run migrations before adding the admin user

### PR DESCRIPTION
**Pull request by Jakob-em**
When the migrations don't run before the admin user is added the following error is thrown when creating a new project with a fresh database:
```
select `collection`, `singleton`, `note`, `sort_field`, `accountability` from `directus_collections` - ER_BAD_FIELD_ERROR: Unknown column 'accountability' in 'field list'
```
This is because the column `accountability` is added by the migrations, but the column is already being used in the getSchema call after the db installation. By running the migrations directly after the installation of the db this problem is fixed.

PS: This caused the e2e tests to fail 🙂

### CODE REVIEW

**Review by nickrum — APPROVED** [review]

**Review by rijkvanzanten — APPROVED** [review]

**Comment by piotr-cz** [comment]
See https://github.com/directus/directus/discussions/13917 for a workaround.

TL;DR
System migrations must be run before schema is applied
